iati/tests/about_functional_tests.py

```diff
@@ -3,29 +3,78 @@
+    admin_browser.find_by_xpath('//div[@class="dropdown-toggle icon icon-arrow-up"]').click()
+    admin_browser.find_by_text('Publish').click()
+
+
+def create_about_child_page(admin_browser, page_type, page_title):
+    """Create a child page in the CMS."""
+    navigate_to_about_cms(admin_browser)
+    admin_browser.find_by_text('Add child page').click()
+    admin_browser.find_by_text(page_type).click()
+    enter_page_content(admin_browser, page_type, page_title)
+    publish_page(admin_browser)
+
+
+def view_live_page(admin_browser, page_title):
+    """Navigate to the published page on the site."""
+    admin_browser.find_by_text(page_title).mouse_over()
+    admin_browser.find_by_text('View live').click()
+
+
 @pytest.mark.django_db()
 class TestAboutChildPageCreation():
     """A container for tests to check the ability to create About child pages."""
 
     @pytest.mark.parametrize('child_page', [
-        {'page_type': 'About sub page', 'title': 'test sub page'},
-        {'page_type': 'Case study index page', 'title': 'test case study index page'},
-        {'page_type': 'History page', 'title': 'test history page'},
-        {'page_type': 'People page', 'title': 'test people page'}
+        ABOUT_SUB_PAGE,
+        CASE_STUDY_INDEX_PAGE,
+        HISTORY_PAGE,
+        PEOPLE_PAGE
     ])
     def test_can_create_about_child_pages(self, admin_browser, child_page):
         """Check that when an about child page is created it appears in the website."""
-        admin_browser.click_link_by_text('Pages')
-        admin_browser.find_by_xpath('//*[@class="icon icon-arrow-right "]').click()
-        admin_browser.find_by_text('About').click()
-        admin_browser.find_by_text('Add child page').click()
-        admin_browser.find_by_text(child_page['page_type']).click()
-        admin_browser.find_by_text('English').click()
-        admin_browser.fill('title_en', child_page['title'])
-        admin_browser.find_by_text('Promote').click()
-        admin_browser.fill('slug_en', slugify(child_page['title']))
-        admin_browser.find_by_xpath('//*[@class="dropdown-toggle icon icon-arrow-up"]').click()
-        admin_browser.find_by_text('Publish').click()
-        admin_browser.find_by_text(child_page['title']).mouse_over()
-        admin_browser.find_by_text('View live').click()
+        create_about_child_page(admin_browser, child_page['page_type'], child_page['title'])
+        view_live_page(admin_browser, child_page['title'])
         assert admin_browser.is_text_present(child_page['title'])
+
+
+@pytest.mark.django_db()
+class TestCaseStudyIndexChildPageCreation():
+    """A container for tests to check the ability to create Case Study pages."""
+
+    def setup_case_study_index_page(self, admin_browser):
+        """Create a Case Study Index page as a child of the About page."""
+        create_about_child_page(admin_browser, CASE_STUDY_INDEX_PAGE['page_type'], CASE_STUDY_INDEX_PAGE['title'])
```

Comment:
DB is transactional for the whole test suite, but not per function, at the moment. As a result this line tries to create a page with a slug identical to the one made in the test_can_create_about_child_pages function above and throws a 500 error.